[2023-01-21 09:00:03,228: INFO/MainProcess] beat: Starting...
[2023-01-21 09:00:03,242: ERROR/MainProcess] Removing schedule tiktok for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:00:03,257: ERROR/MainProcess] Removing schedule upload-schedule-15 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:00:03,266: ERROR/MainProcess] Removing schedule upload-schedule-16 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:00:03,275: ERROR/MainProcess] Removing schedule upload-schedule-17 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:00:03,284: ERROR/MainProcess] Removing schedule upload-schedule-18 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:00:03,292: ERROR/MainProcess] Removing schedule upload-schedule-19 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:01:43,364: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:01:43,372: ERROR/MainProcess] Removing schedule upload-schedule-20 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:01:48,392: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:01:48,399: ERROR/MainProcess] Removing schedule upload-schedule-21 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:01:48,417: ERROR/MainProcess] Removing schedule upload-schedule-22 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:03:08,472: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:03:08,477: ERROR/MainProcess] Removing schedule upload-schedule-23 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:03:13,484: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:03:13,488: ERROR/MainProcess] Removing schedule upload-schedule-24 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:07:02,608: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:07:02,615: ERROR/MainProcess] Removing schedule upload-schedule-25 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 09:07:07,625: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 09:07:07,633: ERROR/MainProcess] Removing schedule upload-schedule-26 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 10:43:46,231: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 10:43:46,248: ERROR/MainProcess] Removing schedule upload-schedule-27 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 10:46:41,350: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 10:46:41,473: ERROR/MainProcess] Removing schedule upload-schedule-28 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 10:47:06,485: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 10:47:06,497: ERROR/MainProcess] Removing schedule upload-schedule-29 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 10:47:41,521: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 10:47:41,536: ERROR/MainProcess] Removing schedule upload-schedule-30 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:05:19,563: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:05:19,578: ERROR/MainProcess] Removing schedule upload-schedule-31 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:06:34,589: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:06:34,605: ERROR/MainProcess] Removing schedule upload-schedule-32 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:07:49,660: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:07:49,667: ERROR/MainProcess] Removing schedule upload-schedule-33 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:08:04,679: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:08:04,693: ERROR/MainProcess] Removing schedule upload-schedule-34 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:09:24,714: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:09:24,730: ERROR/MainProcess] Removing schedule upload-schedule-35 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:09:59,761: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:09:59,781: ERROR/MainProcess] Removing schedule upload-schedule-36 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:13:00,204: INFO/MainProcess] beat: Starting...
[2023-01-21 12:14:55,477: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 12:14:55,483: ERROR/MainProcess] Removing schedule upload-schedule-37 for argument deseralization error: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\django_celery_beat\schedulers.py", line 61, in __init__
    self.args = loads(model.args or '[]')
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\kombu\utils\json.py", line 88, in loads
    return _loads(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\scott\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2023-01-21 12:18:00,786: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:00:45,210: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:01:15,289: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:04:05,713: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:14:18,889: INFO/MainProcess] Scheduler: Sending due task upload-schedule-42 (testing.tasks.test_process)
[2023-01-21 13:14:21,130: INFO/MainProcess] Scheduler: Sending due task upload-schedule-41 (testing.tasks.test_process)
[2023-01-21 13:14:51,203: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:14:56,354: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:15:00,003: INFO/MainProcess] Scheduler: Sending due task upload-schedule-43 (testing.tasks.test_process)
[2023-01-21 13:16:15,183: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:16:20,272: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:19:14,637: INFO/MainProcess] Scheduler: Sending due task upload-schedule-45 (testing.tasks.test_process)
[2023-01-21 13:19:14,644: INFO/MainProcess] Scheduler: Sending due task upload-schedule-44 (testing.tasks.test_process)
[2023-01-21 13:20:04,755: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 13:20:09,882: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:00:11,551: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:00:57,355: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:06:28,069: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:12:03,721: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:42:47,513: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:43:47,656: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:44:00,008: INFO/MainProcess] Scheduler: Sending due task upload-schedule-50 (testing_func)
[2023-01-21 20:48:30,717: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:48:35,803: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:48:50,869: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:50:00,000: INFO/MainProcess] Scheduler: Sending due task upload-schedule-51 (testing func)
[2023-01-21 20:50:00,042: ERROR/MainProcess] Message Error: Couldn't apply scheduled task upload-schedule-51: test_process() missing 1 required positional argument: 'objPk'
['  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 196, in _run_module_as_main\n    return _run_code(code, main_globals, None,\n', '  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 86, in _run_code\n    exec(code, run_globals)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\Scripts\\celery.exe\\__main__.py", line 7, in <module>\n    sys.exit(main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\__main__.py", line 15, in main\n    sys.exit(_main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\celery.py", line 217, in main\n    return celery(auto_envvar_prefix="CELERY")\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1130, in __call__\n    return self.main(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1055, in main\n    rv = self.invoke(ctx)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 760, in invoke\n    return __callback(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\decorators.py", line 26, in new_func\n    return f(get_current_context(), *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\base.py", line 134, in caller\n    return f(ctx, *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\beat.py", line 72, in beat\n    return beat().run()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 77, in run\n    self.start_scheduler()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 105, in start_scheduler\n    service.start()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 645, in start\n    interval = self.scheduler.tick()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 357, in tick\n    self.apply_entry(entry, producer=self.producer)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 290, in apply_entry\n    exc, traceback.format_stack(), exc_info=True)\n']
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
TypeError: test_process() missing 1 required positional argument: 'objPk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 287, in apply_entry
    result = self.apply_async(entry, producer=producer, advance=False)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 412, in apply_async
    reraise(SchedulingError, SchedulingError(
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\exceptions.py", line 108, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
celery.beat.SchedulingError: Couldn't apply scheduled task upload-schedule-51: test_process() missing 1 required positional argument: 'objPk'
[2023-01-21 20:55:55,815: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:56:00,866: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:57:00,975: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 20:57:16,034: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:00:31,377: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:01:00,003: INFO/MainProcess] Scheduler: Sending due task upload-schedule-54 (testing func)
[2023-01-21 21:01:00,004: ERROR/MainProcess] Message Error: Couldn't apply scheduled task upload-schedule-54: test_process() missing 1 required positional argument: 'objPk'
['  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 196, in _run_module_as_main\n    return _run_code(code, main_globals, None,\n', '  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 86, in _run_code\n    exec(code, run_globals)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\Scripts\\celery.exe\\__main__.py", line 7, in <module>\n    sys.exit(main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\__main__.py", line 15, in main\n    sys.exit(_main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\celery.py", line 217, in main\n    return celery(auto_envvar_prefix="CELERY")\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1130, in __call__\n    return self.main(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1055, in main\n    rv = self.invoke(ctx)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 760, in invoke\n    return __callback(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\decorators.py", line 26, in new_func\n    return f(get_current_context(), *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\base.py", line 134, in caller\n    return f(ctx, *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\beat.py", line 72, in beat\n    return beat().run()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 77, in run\n    self.start_scheduler()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 105, in start_scheduler\n    service.start()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 645, in start\n    interval = self.scheduler.tick()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 357, in tick\n    self.apply_entry(entry, producer=self.producer)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 290, in apply_entry\n    exc, traceback.format_stack(), exc_info=True)\n']
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
TypeError: test_process() missing 1 required positional argument: 'objPk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 287, in apply_entry
    result = self.apply_async(entry, producer=producer, advance=False)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 412, in apply_async
    reraise(SchedulingError, SchedulingError(
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\exceptions.py", line 108, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
celery.beat.SchedulingError: Couldn't apply scheduled task upload-schedule-54: test_process() missing 1 required positional argument: 'objPk'
[2023-01-21 21:02:35,239: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:02:40,302: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:04:45,567: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:07:00,000: INFO/MainProcess] Scheduler: Sending due task upload-schedule-56 (testing func)
[2023-01-21 21:07:00,001: ERROR/MainProcess] Message Error: Couldn't apply scheduled task upload-schedule-56: test_process() missing 1 required positional argument: 'objPk'
['  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 196, in _run_module_as_main\n    return _run_code(code, main_globals, None,\n', '  File "C:\\Users\\scott\\AppData\\Local\\Programs\\Python\\Python310\\lib\\runpy.py", line 86, in _run_code\n    exec(code, run_globals)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\Scripts\\celery.exe\\__main__.py", line 7, in <module>\n    sys.exit(main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\__main__.py", line 15, in main\n    sys.exit(_main())\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\celery.py", line 217, in main\n    return celery(auto_envvar_prefix="CELERY")\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1130, in __call__\n    return self.main(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1055, in main\n    rv = self.invoke(ctx)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\core.py", line 760, in invoke\n    return __callback(*args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\click\\decorators.py", line 26, in new_func\n    return f(get_current_context(), *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\base.py", line 134, in caller\n    return f(ctx, *args, **kwargs)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\bin\\beat.py", line 72, in beat\n    return beat().run()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 77, in run\n    self.start_scheduler()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\apps\\beat.py", line 105, in start_scheduler\n    service.start()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 645, in start\n    interval = self.scheduler.tick()\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 357, in tick\n    self.apply_entry(entry, producer=self.producer)\n', '  File "C:\\Users\\scott\\.virtualenvs\\tiktok_uploader-PCpzNl2z\\lib\\site-packages\\celery\\beat.py", line 290, in apply_entry\n    exc, traceback.format_stack(), exc_info=True)\n']
Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
TypeError: test_process() missing 1 required positional argument: 'objPk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 287, in apply_entry
    result = self.apply_async(entry, producer=producer, advance=False)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 412, in apply_async
    reraise(SchedulingError, SchedulingError(
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\exceptions.py", line 108, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\beat.py", line 404, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "C:\Users\scott\.virtualenvs\tiktok_uploader-PCpzNl2z\lib\site-packages\celery\app\task.py", line 540, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
celery.beat.SchedulingError: Couldn't apply scheduled task upload-schedule-56: test_process() missing 1 required positional argument: 'objPk'
[2023-01-21 21:30:17,854: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:30:22,926: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:31:00,005: INFO/MainProcess] Scheduler: Sending due task upload-schedule-57 (testing_func)
[2023-01-21 21:37:25,798: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:37:30,835: INFO/MainProcess] DatabaseScheduler: Schedule changed.
[2023-01-21 21:39:00,000: INFO/MainProcess] Scheduler: Sending due task upload-schedule-58 (testing_func)
